access/pagerduty/bot.go

```diff
@@ -39,90 +40,112 @@ func init() {
 	}
 }
 
-// Bot is a wrapper around pd.Client that works with access.Request
+// Bot is a wrapper around resty.Client.
```

Comment:
nit: this is an implementation detail.
prefer doc comments like "Bot is a client for PagerDuty API"